backend/tests/test_schwab_account_value.py

```
@@ -0,0 +1,387 @@
+    _cache_key,
+)
+from app.services.schwab_auth import SchwabAuthCode, SchwabAuthError
+from app.services.schwab_client import SchwabClientError
+
+
+# ---------------------------------------------------------------------------
+# Fixtures
+# ---------------------------------------------------------------------------
+
+
+@pytest.fixture()
```

Ruff (0.15.13)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)